File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex1.java
//Importando Scanner
import java.util.Scanner;

public class Ex1{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	double n1, n2, res;
	
	System.out.println("Subtração entre dois números\n\n");
	
	//Entradas
	System.out.print("Insira o 1° número: ");
	n1 = ent.nextDouble();
	
	System.out.print("Insira o 2° número: ");
	n2 = ent.nextDouble();
	
	//Processamento
	res = n1 - n2;
	
	//Saída
	System.out.println(n1 + " - " + n2 + " = " + res);
		
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex1.c
#include <stdio.h>

main()
{
    //vars
    float n1,n2,res;

    printf("\t\t\tSubtracao entre dois numeros\n\n");

    //Entradas
    printf("Insira o 1o numero: ");
    scanf("%f",&n1);

    printf("Insira o 2o numero: ");
    scanf("%f", &n2);

    //Processamento
    res = n1 - n2;

    //Saída
    printf("%.2f - %.2f = %.2f", n1, n2, res);

}
